Cargo.toml

```diff
@@ -53,7 +53,8 @@ opt-level = 3 # Speed-up the CI
 # This profile is used for prebuilt binaries.
 [profile.release_prebuilt]
 inherits = "release"
-codegen-units = 1
+codegen-units = 1  # Do not split into multiple codegen units, improve performance, may compile slower
+lto = true         # Enables Link Time Optimization, enabling more aggressive optimizations across the entire codebase
```

Comment:
Just curious, how much longer is the build time?

Reply:
```
sydh@sydh-Vostro-3425:~/dev/massa6$ cargo clean 
     Removed 54190 files, 54.3GiB total
sydh@sydh-Vostro-3425:~/dev/massa6$ time cargo build 
...
real	5m17,706s
user	17m56,969s
sys	2m3,156s
```

versus

```
cargo clean 
     Removed 10597 files, 11.8GiB total
sydh@sydh-Vostro-3425:~/dev/massa6$ time cargo build --profile release_prebuilt
... 
real	6m56,728s
user	9m24,421s
sys	1m13,582s
```

Reply:
Thank you, I thought it would have been a lot slower.